src/drawing/DrawingSVG.tsx

```diff
@@ -1,5 +1,7 @@
-import React from 'react';
+import React, { useRef } from 'react';
```

Comment:
We typically use this as `React.useRef`, since we've already imported the top-level object.